examples/multi_backend/src/multi_backend.cr

```diff
@@ -0,0 +1,48 @@
+require "turbo/cable"
+require "cable/backend/nats"
+require "cable/backend/redis/backend"
+
+module AppCable
+  class Connection < Cable::Connection
+    identified_by id
+
+    getter id = UUID.random.to_s
+
+    def connect
+    end
+  end
+end
+
+Cable.configure do |settings|
+  settings.route = "/cable" # the URL your JS Client will connect
+  # settings.url = "redis:///"
+  # settings.url = ENV.fetch("NATS_URL", "nats:///")
+  settings.url = ENV.fetch("CABLE_BACKEND_URL", "redis:///")
+end
```

Comment:
Notice that in the settings for the demo we aren't actually setting `backend_class` anywhere. Which backend is used is based entirely on `url`.